production.ini

```diff
@@ -513,14 +513,19 @@ autostart_celery_notify_beat = true
 autostart_source_reader = true
 autostart_changes_router = true
 autostart_pserve = false
-autostart_nodesass = false
 autostart_gulp = false
 autostart_webpack = false
 autostart_uwsgi = true
-autostart_metrics_server = false
-autostart_metrics_server = false
-autostart_edgesense_server = false
 autostart_urlmetadata = true
+autostart_elasticsearch = true
+autostart_maintenance = false
+sup_log_backups_celery_notify_beat = 0
+sup_log_backups_gulp = 0
+sup_log_backups_webpack = 0
+sup_log_backups_maintenance = 0
+sup_log_backups_uwsgi = 20
+sup_log_backups_pserve = 2
+sup_log_size_uwsgi_err = 4000000
```

Comment:
Comments of the numbers here won't hurt future devs from digging in supervisor docs to understand that this is in bytes, so this particular is about ~ 3.8 MB.